stellar/pause.js

```diff
@@ -0,0 +1,59 @@
+const { Contract } = require('@stellar/stellar-sdk');
+const { Command, Option } = require('commander');
+const { getWallet, broadcast, addBaseOptions } = require('./utils');
+const { loadConfig, printInfo } = require('../evm/utils');
+const { getChainConfig } = require('../common');
+require('./cli-utils');
+
+async function processCommand(options, _, chain) {
+    const wallet = await getWallet(chain, options);
+
+    const contract = new Contract(options.address || chain.contracts?.pausable_contract?.address);
```

Comment:
```suggestion
    const contract = new Contract(options.address || chain.contracts?[contractName].address);
```
should use contract name input